webpack/scenes/ContentViews/Details/ContentViewDetailSelectors.js

```diff
@@ -13,3 +13,6 @@ export const selectCVDetailStatus =
 
 export const selectCVDetailError =
   (state, cvId) => selectAPIError(state, `${CONTENT_VIEWS_KEY}_${cvId}`);
+
+
+export const selectCVUpdating = state => state.katello?.contentViewDetails?.updating;
```

Comment:
Optional chaining party :tada: 

Since this returns a boolean, prefer naming like `selectIsCVUpdating`